crates/precompile/src/bls12_381_utils.rs

```diff
@@ -17,43 +13,3 @@ pub fn msm_required_gas(k: usize, discount_table: &[u16], multiplication_cost: u
 
     (k as u64 * discount * multiplication_cost) / MSM_MULTIPLIER
 }
-
-pub fn bls12_381_precompiles_not_supported() -> Vec<PrecompileWithAddress> {
-    vec![
-        PrecompileWithAddress(G1_ADD_ADDRESS, |_, _| {
-            Err(PrecompileError::Fatal(
-                "no_std is not supported for BLS12-381 precompiles".into(),
-            ))
-        }),
-        PrecompileWithAddress(G1_MSM_ADDRESS, |_, _| {
-            Err(PrecompileError::Fatal(
-                "no_std is not supported for BLS12-381 precompiles".into(),
-            ))
-        }),
-        PrecompileWithAddress(G2_ADD_ADDRESS, |_, _| {
-            Err(PrecompileError::Fatal(
-                "no_std is not supported for BLS12-381 precompiles".into(),
-            ))
-        }),
-        PrecompileWithAddress(G2_MSM_ADDRESS, |_, _| {
-            Err(PrecompileError::Fatal(
-                "no_std is not supported for BLS12-381 precompiles".into(),
-            ))
-        }),
```

Comment:
Deleted this because arkworks should support no_std -- I guess one thing to note is that to use no_std, you would need to do `cargo b --no-default-features --features arkworks-bls12-381` , no-default-features to remove `std` and arkworks-bls12-381 to switch to arkworks instead of blst.

There is an asymetry here I guess between bn128 which uses blst by default and bls12-381 which uses arkworks as an optional

Reply:
Have changed it to use blst as optional (happy to change back if needed)